src/ContMechTensors.jl

```diff
@@ -250,20 +223,19 @@ for op in (:zero, :rand)
         @eval begin
             @inline function Base.$op{order, dim, T}(Tt::Type{$TensorType{order, dim, T}})
                 N = n_components($TensorType{order, dim})
-                return $TensorType{order, dim, T, N}($op(SVector{N, T}))
+                return $TensorType{order, dim}($op(SVector{N, T}))
```

Comment:
We rand a SVector of T though? I think it is just for vec. Goes through the func def below.